package.json

```diff
@@ -6,6 +6,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
+    "test": "jest --watch",
```

Comment:
This one is for the watch mode of the test. We also need a normal mode test.

Try to create a new script with the command "jest --colors --config jest.config.js"